CMakeLists.txt

```diff
@@ -34,8 +33,10 @@ find_package(aikido 0.3.0 REQUIRED
     planner_vectorfield
     planner_kunzretimer
     robot
-    rviz
+    robot_ros
     statespace
+    io
```

Comment:
Hmm, was there a reason for moving `io`? I believe the norm is to alphabetize it (in which case, rviz comes before statespace and io after distance)